crates/chainspec/src/spec.rs

```diff
@@ -258,6 +258,13 @@ impl From<ForkBaseFeeParams> for BaseFeeParamsKind {
     }
 }
 
+/// The type for specify which is kind of engine api
+#[derive(Debug)]
+pub enum EngineApiKind {
```

Comment:
this should be moved to:

https://github.com/paradigmxyz/reth/blob/06e7afbd2d898b6f95ff7b757f53327d189e2048/crates/engine/tree/src/engine.rs

Reply:
@mattsse, what does you means by `defaut`?